flake.nix

```diff
@@ -11,7 +11,7 @@
       inputs.nixpkgs.follows = "nixpkgs";
       inputs.spectrum.follows = "";
     }; # Currently using this fork since the upstream seems to be causing an issue
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.05";
```

Comment:
For my own edification....Is this change solely to provide more confidence around package stability?  I'm curious if something happened that prompted needing to tie to 24.05, or if this is just a better "sane default".

Reply:
good question. It was both: I needed new nixos module options that werent available in nixos-unstable from ~Oct 2023 (when the lock was last generated) and to bump to a version that was currently stable to avoid any breakage.